[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:53,157][4188:6632][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:53,158][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Max_Pastdue_Loan_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:53,158][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:39:54,346][4188:6632][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:39:54,608][4188:6632][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:39:54,686][4188:6632][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:56,014][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_Acct_Unpaidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:56,015][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:56,073][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:57,809][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Max_Pastdue_Creditcard_12Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:57,809][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_12Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:57,870][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:59,628][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Num_Pastdue_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:59,629][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:39:59,689][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:01,418][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：Max_Pastdue_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:01,418][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:01,479][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:03,190][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Balance_Paid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:03,190][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_Paid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:03,250][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:40:03,574][4188:6632][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:05,027][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：7/96，正在处理变量：Tot_Amount_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:05,027][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:05,086][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage55" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:06,807][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：8/96，正在处理变量：Max_Pastdue_Creditcard_24Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:06,808][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_24Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:06,868][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage64" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:08,596][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：9/96，正在处理变量：Max_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:08,597][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:08,656][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage73" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:10,366][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：10/96，正在处理变量：Num_DebitCard_Pastdue_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:10,366][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_DebitCard_Pastdue_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:10,425][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage82" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:12,354][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：11/96，正在处理变量：Num_Acct_Unpaidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:12,354][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:12,412][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage91" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:14,175][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：12/96，正在处理变量：Num_Acct_Paidoff_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:14,176][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:14,235][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage100" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:15,920][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：13/96，正在处理变量：Avg_CL_DebitCard_6mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:15,920][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_6mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:15,981][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage109" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:22,866][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：14/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:25,629][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:44,001][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage118" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:46,059][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：15/96，正在处理变量：Tot_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:46,060][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:50,641][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage127" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:52,574][4188:6632][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：16/96，正在处理变量：Num_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:52,574][4188:6632][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:40:53,240][4188:6632][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage136" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:00,058][8984:4656][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:00,058][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:00,058][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:54:00,593][8984:4656][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:54:00,614][8984:4656][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:54:00,665][8984:4656][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:54:00,737][8984:4656][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:02,076][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_DebitCard_Pastdue_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:02,077][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_DebitCard_Pastdue_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:02,136][8984:4656][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:03,810][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Num_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:03,811][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:03,869][8984:4656][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:05,572][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:05,572][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:05,630][8984:4656][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:07,282][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：Num_Acct_Paidoff_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:07,283][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:07,342][8984:4656][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:09,001][8984:4656][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Max_Pastdue_Loan_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:09,001][8984:4656][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:54:09,060][8984:4656][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:21,648][8764:9140][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:21,649][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Num_DebitCard_ChargeOff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:21,649][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_DebitCard_ChargeOff是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:55:22,025][8764:9140][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:55:22,079][8764:9140][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:55:22,145][8764:9140][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:23,447][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_Acct_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:23,447][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:23,505][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:25,142][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Max_Pastdue_Creditcard_12Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:25,142][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_12Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:25,200][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:26,894][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:26,894][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:26,952][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 09:55:27,224][8764:9140][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:28,663][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：DebitCard_Pastdue_Tot_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:28,663][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Tot_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:28,722][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:30,387][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Num_Acct_Paidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:30,388][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:30,446][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:32,094][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：7/96，正在处理变量：Num_Acct_Unpaidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:32,095][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:32,154][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage55" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:33,855][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：8/96，正在处理变量：Num_Acct_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:33,855][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:33,915][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage64" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:35,570][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：9/96，正在处理变量：Num_Pastdue_Creditcard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:35,570][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:35,630][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage73" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:37,324][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：10/96，正在处理变量：Tot_Amount_Pastdue_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:37,324][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Pastdue_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:37,384][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage82" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:39,158][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：11/96，正在处理变量：Max_Pastdue_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:39,158][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:39,217][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage91" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:40,931][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：12/96，正在处理变量：Avg_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:40,931][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:40,991][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage100" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:42,750][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：13/96，正在处理变量：DebitCard_Pastdue_Max_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:42,751][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Max_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:42,811][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage109" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:44,642][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：14/96，正在处理变量：Max_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:44,642][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:44,702][8764:9140][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage118" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:48,927][8764:9140][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：15/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 09:55:48,927][8764:9140][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:08,177][8348:6480][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:08,177][8348:6480][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Num_Pastdue_Creditcard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:08,177][8348:6480][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:16:09,005][8348:6480][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:16:09,068][8348:6480][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:16:09,144][8348:6480][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:10,491][8348:6480][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Unpaid_Balance_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:10,491][8348:6480][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Unpaid_Balance_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:16:10,551][8348:6480][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:21:28,895][5700:6380][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:21:28,896][5700:6380][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Balance_Paid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:21:28,896][5700:6380][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_Paid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:21:29,302][5700:6380][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:21:29,325][5700:6380][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:21:29,378][5700:6380][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:21:29,460][5700:6380][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:12,425][7148:5552][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:12,425][7148:5552][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:12,425][7148:5552][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:23:12,827][7148:5552][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:23:12,848][7148:5552][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:23:12,901][7148:5552][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:23:12,981][7148:5552][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:14,359][7148:5552][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Max_Pastdue_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:14,359][7148:5552][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:23:14,418][7148:5552][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:32:23,238][8024:2948][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:32:23,239][8024:2948][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Max_Pastdue_Creditcard_24Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:32:23,239][8024:2948][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_24Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:32:23,634][8024:2948][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:32:23,690][8024:2948][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:32:23,767][8024:2948][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:30,817][9020:7304][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:30,818][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Num_Acct_Paidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:30,818][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:34:31,204][9020:7304][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:34:31,262][9020:7304][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:34:31,344][9020:7304][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:32,675][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Max_Pastdue_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:32,675][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:32,738][9020:7304][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:34,424][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:34,424][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:34,483][9020:7304][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:36,130][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Avg_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:36,130][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:36,192][9020:7304][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 10:34:36,462][9020:7304][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:37,909][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：Tot_Balance_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:37,910][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Balance_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:37,968][9020:7304][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:39,663][9020:7304][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:39,664][9020:7304][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 10:34:39,724][9020:7304][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:20,663][8492:2548][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:20,664][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Num_Acct_Unpaidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:20,664][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:04:21,056][8492:2548][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:04:21,113][8492:2548][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:04:21,186][8492:2548][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:22,494][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:22,495][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:22,554][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:24,227][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Tot_Balance_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:24,227][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Balance_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:24,286][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:04:24,597][8492:2548][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:25,986][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Tot_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:25,986][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:26,046][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:27,779][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：Max_Pastdue_Loan_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:27,779][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:27,839][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:29,511][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:29,511][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:29,572][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:31,314][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：7/96，正在处理变量：Num_Pastdue_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:31,314][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:31,372][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage55" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:33,081][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：8/96，正在处理变量：Num_Acct_Paidoff_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:33,082][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:33,143][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage64" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:34,822][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：9/96，正在处理变量：Tot_Amount_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:34,823][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:34,881][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage73" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:36,615][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：10/96，正在处理变量：DebitCard_Pastdue_Tot_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:36,616][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Tot_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:36,878][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage82" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:38,982][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：11/96，正在处理变量：Num_Acct_Unpaidoff_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:38,983][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:39,484][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage91" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:41,307][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：12/96，正在处理变量：Num_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:41,307][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:41,366][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage100" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:44,760][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：13/96，正在处理变量：Num_Acct_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:44,882][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:04:56,248][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage109" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:00,131][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：14/96，正在处理变量：Tot_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:00,131][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:08,868][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage118" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:15,386][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：15/96，正在处理变量：Tot_Balance_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:15,386][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Balance_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:24,306][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage127" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:27,573][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：16/96，正在处理变量：Max_Pastdue_Creditcard_12Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:27,573][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_12Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:05:32,510][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage136" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:06:03,569][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：17/96，正在处理变量：Num_Pastdue_Creditcard_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:06:03,727][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:06:07,646][8492:2548][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage145" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:06:13,773][8492:2548][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：18/96，正在处理变量：Num_Acct_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:06:13,773][8492:2548][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:11:58,633][9508:7132][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:11:58,633][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Max_Pastdue_Creditcard_24Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:11:58,633][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_24Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:11:59,998][9508:7132][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:12:00,054][9508:7132][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:12:00,131][9508:7132][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:01,470][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_Acct_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:01,471][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:01,531][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:03,187][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Avg_CL_DebitCard_6mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:03,188][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_6mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:03,250][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:12:03,520][9508:7132][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:04,949][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Tot_Num_Loantypes
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:04,949][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Num_Loantypes是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:05,008][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:06,677][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：DebitCard_Pastdue_Tot_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:06,677][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Tot_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:06,735][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:08,409][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Num_Acct_Paidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:08,410][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:08,467][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:10,171][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：7/96，正在处理变量：Avg_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:10,171][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:10,231][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage55" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:11,921][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：8/96，正在处理变量：Tot_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:11,921][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:11,979][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage64" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:13,705][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：9/96，正在处理变量：Balance_Paid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:13,705][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_Paid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:13,762][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage73" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:15,452][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：10/96，正在处理变量：Avg_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:15,452][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:15,511][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage82" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:17,208][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：11/96，正在处理变量：Tot_Amount_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:17,208][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:17,267][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage91" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:18,951][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：12/96，正在处理变量：Num_DebitCard_Pastdue_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:18,951][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_DebitCard_Pastdue_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:19,011][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage100" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:20,734][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：13/96，正在处理变量：Num_Acct_Unpaidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:20,734][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:20,798][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage109" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:22,531][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：14/96，正在处理变量：Num_Loan_Due_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:22,531][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Due_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:22,591][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage118" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:24,364][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：15/96，正在处理变量：Num_Loan_Due_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:24,364][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Due_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:24,425][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage127" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:26,190][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：16/96，正在处理变量：Num_Acct_Unpaidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:26,190][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:26,249][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage136" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:28,006][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：17/96，正在处理变量：Num_Acct_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:28,006][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:28,068][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage145" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:43,964][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：18/96，正在处理变量：Max_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:45,208][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:12:57,353][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage154" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:00,265][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：19/96，正在处理变量：DebitCard_Pastdue_Max_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:00,265][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Max_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:00,325][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage163" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:05,163][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：20/96，正在处理变量：Percentage_Paid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:05,163][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Percentage_Paid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:05,704][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage172" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 15:13:21,729][9508:7132][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=45.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:30,845][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：21/96，正在处理变量：Num_Acct_Paidoff_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:13:30,846][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:13,859][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage181" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:26,470][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：22/96，正在处理变量：Tot_Amount_Pastdue_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:26,471][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Pastdue_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:29,113][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage190" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:34,539][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：23/96，正在处理变量：Unpaid_Balance_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:14:34,540][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Unpaid_Balance_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:31,739][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage199" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:34,594][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：24/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:34,594][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:34,905][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage208" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:37,080][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：25/96，正在处理变量：Num_Pastdue_Creditcard_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:37,080][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:37,833][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage217" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:40,486][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：26/96，正在处理变量：Max_Pastdue_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:40,487][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:40,855][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage226" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:47,110][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：27/96，正在处理变量：Num_Acct_Pastdue_Loan_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:15:47,110][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Pastdue_Loan_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:04,772][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage235" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:06,944][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：28/96，正在处理变量：Tot_Acct_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:06,944][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Acct_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:07,221][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage244" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:09,324][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：29/96，正在处理变量：Tot_Balance_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:09,325][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Balance_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:09,516][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage253" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:23,420][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：30/96，正在处理变量：Num_Pastdue_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:23,421][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:51,888][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage262" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:54,873][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：31/96，正在处理变量：Num_Pastdue_Loan_24Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:54,873][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Loan_24Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:16:55,497][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage271" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:45,609][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：32/96，正在处理变量：Num_Acct_Paidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:45,610][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:45,743][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage280" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:47,976][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：33/96，正在处理变量：Max_Pastdue_Loan_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:47,976][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:17:56,062][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage289" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:13,673][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：34/96，正在处理变量：Num_Acct_Paidoff_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:13,674][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:37,027][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage298" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:39,115][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：35/96，正在处理变量：Num_Acct_Unpaidoff_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:39,115][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:39,806][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage307" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:42,773][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：36/96，正在处理变量：Tot_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:42,773][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:43,005][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage316" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:45,497][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：37/96，正在处理变量：Max_Pastdue_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:45,497][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:45,768][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage325" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:51,688][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：38/96，正在处理变量：Num_Pastdue_Creditcard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:18:51,689][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:03,910][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage334" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:19,676][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：39/96，正在处理变量：Avg_Amount_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:19,677][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:41,401][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage343" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:53,891][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：40/96，正在处理变量：Num_DebitCard_ChargeOff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:19:53,892][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_DebitCard_ChargeOff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:20:10,350][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage352" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:20:16,231][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：41/96，正在处理变量：Num_Acct_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:20:16,232][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:20:57,548][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage361" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:21:04,613][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：42/96，正在处理变量：Max_Pastdue_Creditcard_12Mon
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:21:04,614][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Max_Pastdue_Creditcard_12Mon是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:21:49,914][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage370" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:00,696][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：43/96，正在处理变量：Tot_Balance_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:00,696][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Balance_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:17,392][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage379" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:21,332][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：44/96，正在处理变量：Amountdue_to_OrigBalance_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:21,333][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Amountdue_to_OrigBalance_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:49,171][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage388" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:52,977][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：45/96，正在处理变量：Tot_Amount_Due_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:22:53,344][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Due_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:15,878][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage397" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:24,201][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：46/96，正在处理变量：CreditCard_Min_MOB
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:24,201][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
CreditCard_Min_MOB是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:41,987][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage406" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:52,265][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：47/96，正在处理变量：Monthly_Payment_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:23:52,265][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Monthly_Payment_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:24:11,644][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage415" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:27:41,852][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：48/96，正在处理变量：ContractValue_Unpaid_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:27:41,852][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
ContractValue_Unpaid_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:27:56,255][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage424" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:13,910][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：49/96，正在处理变量：Repayment_Percentage_Debitcard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:13,910][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Repayment_Percentage_Debitcard是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:29,146][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage433" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:33,287][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：50/96，正在处理变量：Working.Years
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:33,287][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Working.Years是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:33,749][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage442" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:36,143][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：51/96，正在处理变量：Tot_Amount_Paid_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:36,143][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Paid_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:36,382][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage451" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:38,864][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：52/96，正在处理变量：Age
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:38,864][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Age是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:39,060][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage460" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:47,026][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：53/96，正在处理变量：Tot_Amount_Due_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:28:47,027][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Due_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:10,044][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage469" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:16,377][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：54/96，正在处理变量：Payment_to_RemainBalance_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:16,377][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Payment_to_RemainBalance_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:19,155][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage478" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:21,794][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：55/96，正在处理变量：Monthly_Payment_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:21,795][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Monthly_Payment_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:21,924][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage487" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:28,648][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：56/96，正在处理变量：Total_Amount_Paid_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:28,649][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Total_Amount_Paid_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:47,619][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage496" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:51,995][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：57/96，正在处理变量：Num_Loan_Open
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:29:51,995][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Open是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:08,852][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage505" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:18,546][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：58/96，正在处理变量：Total_Amount_Due_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:18,546][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Total_Amount_Due_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:46,559][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage514" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:51,547][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：59/96，正在处理变量：Monthly.Income
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:30:51,548][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Monthly.Income是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:13,133][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage523" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:17,594][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：60/96，正在处理变量：Total_Amount_Due_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:17,594][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Total_Amount_Due_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:18,934][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage532" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:35,521][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：61/96，正在处理变量：Tot_DebitCard_Due_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:35,522][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_DebitCard_Due_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:31:57,682][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage541" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:03,820][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：62/96，正在处理变量：Balance_Unpaid_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:03,820][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_Unpaid_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:23,056][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage550" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:43,135][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：63/96，正在处理变量：Total_Amount_Due_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:43,135][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Total_Amount_Due_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:32:59,390][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage559" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:36:10,116][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：64/96，正在处理变量：Percentage_Paid_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:36:10,116][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Percentage_Paid_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:36:36,559][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage568" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:36:57,088][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：65/96，正在处理变量：Total_Amount_Paid_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:36:57,190][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Total_Amount_Paid_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:19,724][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage577" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:21,941][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：66/96，正在处理变量：Min_MOB_AllLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:21,942][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Min_MOB_AllLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:24,992][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage586" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:28,704][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：67/96，正在处理变量：Amountdue_to_OrigBalance_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:30,366][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Amountdue_to_OrigBalance_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:31,129][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage595" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:33,418][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：68/96，正在处理变量：Tot_Amount_Due_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:33,434][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Due_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:33,739][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage604" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:36,709][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：69/96，正在处理变量：Num_Acct_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:36,843][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:37:37,238][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage613" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:38:04,261][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：70/96，正在处理变量：ContractValue_Unpaid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:38:04,261][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
ContractValue_Unpaid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:39:23,046][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage622" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:39:47,874][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：71/96，正在处理变量：Amount_Paid_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:39:47,875][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Amount_Paid_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:14,575][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage631" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:39,367][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：72/96，正在处理变量：Amount_Paid_Mortage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:39,368][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Amount_Paid_Mortage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:46,873][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage640" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[31m/--------------------------------------------------------------------------------
[ERROR][2018-09-14 15:40:46,874][9508:7132][UnivariateAnalysisPlot.py:64 plot_preprocess_result]:
为变量Amount_Paid_Mortage画图时发生错误，跳过该变量的画图。
错误信息如下：
not enough free memory for image buffer
画图参数是
bin_names: ['[-1.00, 0.00)', '[0.00, 100000.00)', '[100000.00, 400000.00)', '[400000.00, 1136000.00)', '[1136000.00, 32163801.00)']
count_list: [22643, 409, 2745, 1580, 527]
proportion_list: [0.06796802543832531, 0.03667481662591687, 0.057194899817850635, 0.07721518987341772, 0.13282732447817835]
IV: 0.023211
var_name: Amount_Paid_Mortage
folder: E:/DataBrain/Fosun_Learn/Analysis
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:46,874][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：73/96，正在处理变量：Balance_UnPaid_SecuredLoan_Property
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:47,439][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_UnPaid_SecuredLoan_Property是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:52,189][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage642" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[31m/--------------------------------------------------------------------------------
[ERROR][2018-09-14 15:40:52,227][9508:7132][UnivariateAnalysisPlot.py:64 plot_preprocess_result]:
为变量Balance_UnPaid_SecuredLoan_Property画图时发生错误，跳过该变量的画图。
错误信息如下：
not enough free memory for image buffer
画图参数是
bin_names: ['[-1.00, 0.00)', '[0.00, 574686.00)', '[574686.00, 772827.00)', '[772827.00, 1178334.00)', '[1178334.00, 30332108.00)']
count_list: [10273, 12341, 1763, 1763, 1764]
proportion_list: [0.0632726564781466, 0.06822785835831781, 0.055587067498581964, 0.06693136698808849, 0.11054421768707483]
IV: 0.027350
var_name: Balance_UnPaid_SecuredLoan_Property
folder: E:/DataBrain/Fosun_Learn/Analysis
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:52,228][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：74/96，正在处理变量：Balance_to_LoanAmount
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:52,228][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Balance_to_LoanAmount是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:54,191][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage644" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[31m/--------------------------------------------------------------------------------
[ERROR][2018-09-14 15:40:54,259][9508:7132][UnivariateAnalysisPlot.py:64 plot_preprocess_result]:
为变量Balance_to_LoanAmount画图时发生错误，跳过该变量的画图。
错误信息如下：
not enough free memory for image buffer
画图参数是
bin_names: ['[-1.00, 0.00)', '[0.00, 72.49)', '[72.49, 80.35)', '[80.35, 88.31)', '[88.31, 90.91)', '[90.91, 93.31)', '[93.31, 97.72)', '[97.72, 101.00)']
count_list: [4565, 4668, 2334, 4667, 2334, 2334, 4668, 2334]
proportion_list: [0.04227820372398686, 0.07262210796915167, 0.061696658097686374, 0.07263766873794729, 0.06341045415595545, 0.08483290488431877, 0.07326478149100257, 0.0856898029134533]
IV: 0.048616
var_name: Balance_to_LoanAmount
folder: E:/DataBrain/Fosun_Learn/Analysis
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:54,362][9508:7132][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：75/96，正在处理变量：Avg_CL_DebitCard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:40:54,363][9508:7132][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 15:41:12,302][9508:7132][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage646" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:12,981][9808:2160][UnivariateAnalysisPlot.py:24 plot_preprocess_result]:
开始执行plot_preprocess_result()。
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:12,981][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：1/96，正在处理变量：Tot_Amount_Loan_6Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:12,982][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Amount_Loan_6Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 16:23:13,864][9808:2160][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 16:23:14,067][9808:2160][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=37.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 16:23:14,116][9808:2160][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=43.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans.ttf') with score of 0.050000
--------------------------------------------------------------------------------/[0m
[36m/--------------------------------------------------------------------------------
[DEBUG][2018-09-14 16:23:14,195][9808:2160][font_manager.py:1343 findfont]:
findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=bold:stretch=normal:size=54.0 to DejaVu Sans ('F:\\tools\\anaconda2\\lib\\site-packages\\matplotlib\\mpl-data\\fonts\\ttf\\DejaVuSans-Bold.ttf') with score of 0.000000
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:15,560][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：2/96，正在处理变量：Num_Acct_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:15,560][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:15,617][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage10" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:17,266][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：3/96，正在处理变量：Num_Acct_CommercialLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:17,266][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_CommercialLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:17,323][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage19" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:18,953][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：4/96，正在处理变量：Num_Acct_Paidoff_Mortgage
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:18,953][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Paidoff_Mortgage是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:19,010][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage28" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:20,624][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：5/96，正在处理变量：Num_Acct_Unpaidoff_AutoLoan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:20,625][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Acct_Unpaidoff_AutoLoan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:20,683][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage37" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:22,361][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：6/96，正在处理变量：Tot_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:22,361][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:22,421][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage46" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:24,109][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：7/96，正在处理变量：Unpaid_Balance_Loan
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:24,109][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Unpaid_Balance_Loan是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:24,171][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage55" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:25,811][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：8/96，正在处理变量：DebitCard_Pastdue_Tot_CurrentMo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:25,811][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
DebitCard_Pastdue_Tot_CurrentMo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:25,869][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage64" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:27,553][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：9/96，正在处理变量：Num_Pastdue_Creditcard
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:27,553][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Pastdue_Creditcard是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:27,611][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage73" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:29,374][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：10/96，正在处理变量：Avg_Amount_Loan_12Mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:29,375][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_Amount_Loan_12Mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:29,433][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage82" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:31,138][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：11/96，正在处理变量：Avg_CL_DebitCard_6mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:31,138][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_6mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:31,196][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage91" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:32,934][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：12/96，正在处理变量：Avg_CL_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:32,934][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Avg_CL_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:32,994][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage100" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:34,729][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：13/96，正在处理变量：Num_Loan_Paidoff
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:34,730][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Num_Loan_Paidoff是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:34,790][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage109" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:36,626][9808:2160][UnivariateAnalysisPlot.py:28 plot_preprocess_result]:
总进度：14/96，正在处理变量：Tot_Acct_DebitCard_12mo
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:36,626][9808:2160][UnivariateAnalysisPlot.py:35 plot_preprocess_result]:
Tot_Acct_DebitCard_12mo是数值型变量
--------------------------------------------------------------------------------/[0m
[32m/--------------------------------------------------------------------------------
[INFO][2018-09-14 16:23:36,686][9808:2160][_backend_tk.py:1058 new_figure_manager_given_figure]:
Could not load matplotlib icon: can't use "pyimage118" as iconphoto: not a photo image
--------------------------------------------------------------------------------/[0m
